lighthouse-cli/test/fixtures/offline-ready.html

```diff
@@ -24,18 +24,28 @@ <h4>
 </h4>
 
 <script>
-  if ('serviceWorker' in navigator) {
-    navigator.serviceWorker.register('/offline-ready-sw.js').then(function(registration) {
+  async function registerServiceWorker(qs = '') {
+    if (!('serviceWorker' in navigator)) return
+
+    try {
+      const registration = await navigator.serviceWorker.register(`/offline-ready-sw.js${qs}`)
       console.log('service worker registration complete');
 
       registration.addEventListener('statechange',  e => {
         console.log('sw registration is now', e.target.state);
       });
-    }).catch(function(e) {
+    } catch (e) {
       console.error('service worker is not so cool.', e);
       throw e;
```

Comment:
do we check for this anywhere? Or is the hope that a smoke test will fail and this would be used for debugging.

Maybe we should be pulling in `errors-in-console` into all our smoke tests :)

Reply:
> do we check for this anywhere?

nope

> Or is the hope that a smoke test will fail and this would be used for debugging.

Bingo!

> Maybe we should be pulling in errors-in-console into all our smoke tests :)

Not a bad idea, followup! :D